cfgov/v1/__init__.py

```diff
@@ -35,12 +39,30 @@ def environment(**options):
         'choices_for_page_type': ref.choices_for_page_type,
         'is_blog': ref.is_blog,
         'get_page_state_url': share.get_page_state_url,
+        'external_links': external_links,
     })
     env.filters.update({
         'slugify': slugify,
     })
     return env
 
+EXTERNAL_LINK_PATTERN = '(https?:\/\/(?:www\.)?(?![^\?]*(cfpb|consumerfinance).gov)(?!(content\.)?localhost).*)'
+
+
+def external_links(value):
+    soup = BeautifulSoup(expand_db_html(value.source))
+
+    try:
+        for a in soup('a'):
+            p = re.compile(EXTERNAL_LINK_PATTERN)
```

Comment:
@kave I don't actually know how much less performant a reg ex is, but since this is being called a lot, I thought it might be more performant to do an if statement, like so:

`if 'cfpb.gov' or 'consumerfinance.gov' in a['href']:` 

which should take care of 3 of those cases. we could throw in localhost in there too, or set up an environment variable. I realize that sounds more complex code-wise, so if it doesn't sound like it'd make that much of a difference performance-wise, I think this is fine!